pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -586,22 +586,30 @@ func (rc *reconciler) updateDevicePath(volumesNeedUpdate map[v1.UniqueVolumeName
 }
 
 func getDeviceMountPath(volume *reconstructedVolume) (string, error) {
-	volumeAttacher, err := volume.attachablePlugin.NewAttacher()
-	if volumeAttacher == nil || err != nil {
-		return "", err
-	}
-	deviceMountPath, err :=
-		volumeAttacher.GetDeviceMountPath(volume.volumeSpec)
-	if err != nil {
-		return "", err
-	}
-
+	var (
+		deviceMountPath string
+		err             error
+	)
 	if volume.blockVolumeMapper != nil {
+		// for block volume, we return its global map path
```

Comment:
check `blockVolumeMapper` first because:

- for local volume, AttachableVolumePlugin is not needed and not implemented
- if it's not nil, deviceMountPath returned from `volumeAttacher.GetDeviceMountPath` is overridden